src/Umbraco.Core/Persistence/Repositories/Implement/RelationRepository.cs

```diff
@@ -253,7 +253,7 @@ public IEnumerable<IRelation> GetPagedRelationsByQuery(IQuery<IRelation> query,
             // apply ordering
             ApplyOrdering(ref sql, ordering);
 
-            var pageIndexToFetch = pageIndex + 1;
+            var pageIndexToFetch = pageIndex;
```

Comment:
This shouldn't be changed. The var name `pageIndexToFetch` could prob be changed to `pageNumberToFetch`. When you see a term called "Index" for paging in our codebase it means it's a zero based index, when you see "Number" it means it's a 1 based number. So you should be passing in the zero based index to this method just like all of our other paging methods.